examples/00_tutorials/plot_single_subject_single_run.py

```diff
@@ -107,31 +122,36 @@
 # Formally, we have taken the first design matrix, because the model is
 # implictily meant to for multiple runs.
 from nilearn.plotting import plot_design_matrix
+
```

Comment:
I think it'd be cleaner to move the matplotlib import together with the unlearn import.

Reply:
done